packages/Interface/src/DTK_C_API.h

```diff
@@ -64,6 +64,9 @@ extern bool DTK_is_initialized();
 /*! Finalize DTK */
 extern void DTK_finalize();
 
+/*! Get error message */
+extern int DTK_error( int err, char *msg );
```

Comment:
if we are setting `errno` couldnt we then just have a function that doesn't need to get the error code? I guess this assumes that we immediately call this after a DTK function call.